src/Sarif.Driver/Sdk/CommandBase.cs

```diff
@@ -18,11 +18,9 @@ public abstract class CommandBase
 
         protected IFileSystem FileSystem { get; set; }
 
-        public CommandBase(IFileSystem fileSystem = null)
```

Comment:
>public CommandBase(IFileSystem fileSystem = null) [](start = 8, length = 49)

don't remove this. For now, this is the way we initialize the FileSytem. If you check, that is used in MultifileCommandBase, calling the FileSystem directly.

Reply:
Fixed.

---
In reply to: [564738564](https://github.com/microsoft/sarif-sdk/pull/2252#discussion_r564738564) [](ancestors = 564738564)